pom.xml

```diff
@@ -19,6 +19,10 @@
     </properties>
 
     <repositories>
+        <repository>
+            <id>spigot</id>
+            <url>https://hub.spigotmc.org/nexus/content/repositories/snapshots/</url>
+        </repository>
```

Comment:
This was intentionally excluded, and should not be added in an unrelated PR

Reply:
Surprising as the project will not compile in a new environment without it, but I'll exclude it